instance_skel.js

```diff
@@ -227,6 +246,25 @@ instance.prototype.setFeedbackDefinitions = function (feedbacks) {
 	if (feedbacks === undefined) {
 		self._feedbackDefinitions = {}
 	} else {
+		feedbacks = Object.fromEntries(
+			Object.entries(feedbacks).map(([id, feedback]) => {
+				feedback.options = feedback.options?.map((option) => {
+					if ('isVisible' in option) {
+						if (typeof option.isVisible === 'function') {
+							return {
+								...option,
+								isVisibleFn: option.isVisible.toString(),
```

Comment:
At the various points we do this, we should set `isVisible: undefined` to make sure socket.io doesnt also try to serialize function

Reply:
I didn't think it would be necessary because
- the reason we're "manually" serializing the function is because JSON doesn't support them and Socket.io doesn't do it on our behalf
- on the WebUI we only use `isVisibleFn` and ignore `isVisible`

That said, sure. I'll add. Better to be safe.